### ISSUE TITLE
Missing accessibility labels

### ISSUE THREAD

**Issue by mhameed** [meypod/al-azan#54]
Salaam,
Jazak Allah kheer for releasing this app on fdroid.

I was wondering if you could add accessibility labels for the buttons, so that a blind person using talkback can successfully use the app.

Thank you.

**Comment by meypod** [comment]
Va alaykum assalam

I'm very busy at the moment but I will look into it when I have time

**Comment by meypod** [comment]
I need help on this one, I have little idea about what and where labels are missing and how a blind person might use the app

**Comment by mhameed** [comment]
please see:
https://support.google.com/accessibility/android/answer/7158690?hl=en-GB

the screenreader can usually be enabled via settings -> accessibility -> talkback
but if you have not used it before, it might take some time to get use to.
If you want we can have a video call and we can go through it in more detail.

Anyway, I will try to find the time to look at the code/send a pull request.

Thanks for your time.

**Comment by meypod** [comment]
Thanks for the link

> Anyway, I will try to find the time to look at the code/send a pull request.

much appreciated

**Comment by Tanzin01** [comment]
What's the status of this issue ? 
I can contribute if needed, if Allah swt wishes :)

**Comment by meypod** [comment]
I'm sorry, I'm in middle of a big migration to a yarn monorepo
While I would love contribution, it would make it harder to migrate things
I'll let you know when I finish migrating 
thank you

**Comment by meypod** [comment]
Well mono repo turned out to be a very bad idea for react native, sadly
@Tanzin01 please make a PR whenever you have time, it is appreciated

**Comment by meypod** [comment]
I tried to add all missing labels
please try the latest releases from GitHub if you can (1.17.0)
will release it on fdroid and google play soon

feel free to open the issue again if there's any issue
